qa/os/build.gradle

```diff
@@ -58,7 +58,7 @@ tasks.dependenciesInfo.enabled = false
 tasks.thirdPartyAudit.ignoreMissingClasses()
 
 tasks.register('destructivePackagingTest') {
-  dependsOn 'destructiveDistroTest' 
+  dependsOn 'destructiveDistroTest', 'destructiveBatsTest.plugins', 'destructiveBatsTest.upgrade'
```

Comment:
Would simply using `tasks.withType(BatsTestTask)` work just as well here and be slightly less brittle?

Reply:
While it would work here, it would not work in the later example because only the destructive test is of type BatsTestTask. The latter is a GradleDistroTestTask, which is a wrapper to call into the VM. Given only these two cases remain, and their time should be limited, I'm not sure it is worth further trying to generify this.